Documentation/WritingDocsOfficial/LocalEditing.rst

```diff
@@ -3,111 +3,116 @@
 .. index:: Official documentation; Local editing
 .. _docs-contribute-git-docker:
 
-======================================================
-Workflow #2: "Local editing and rendering with Docker"
-======================================================
+=============================================================
+Workflow #2: Working locally on the documentation with GitHub
```

Comment:
I would avoid "with GitHub". Actuall, I don' work locally with GitHub. I work with PhpStorm and Git.

Reply:
Ack